### PR TITLE
option to enable horizontal dragging when orientation camera is using…

### PR DESCRIPTION
**Pull request by TrevorDev**
… sensor

https://github.com/BabylonJS/Babylon.js/issues/6048

TestPG: #CGQ5QL#1

### CODE REVIEW

File: src/Cameras/Inputs/freeCameraMouseInput.ts [hunk 1]
```diff
@@ -30,7 +30,9 @@ export class FreeCameraMouseInput implements ICameraInput<FreeCamera> {
     private _onMouseMove: Nullable<(e: MouseEvent) => any>;
     private _observer: Nullable<Observer<PointerInfo>>;
     private previousPosition: Nullable<{ x: number, y: number }> = null;
-
+    
+    public _onMouseMoved = new Observable<{ offsetX: number, offsetY: number }>();
```
Comment (deltakosh) [inline src/Cameras/Inputs/freeCameraMouseInput.ts]: this could even be public as other people can use it

Comment (deltakosh) [reply src/Cameras/Inputs/freeCameraMouseInput.ts]: But I would rename it OnPointerMovedObservable

File: src/Cameras/Inputs/freeCameraMouseInput.ts [hunk 2]
```diff
@@ -30,7 +30,9 @@ export class FreeCameraMouseInput implements ICameraInput<FreeCamera> {
     private _onMouseMove: Nullable<(e: MouseEvent) => any>;
     private _observer: Nullable<Observer<PointerInfo>>;
     private previousPosition: Nullable<{ x: number, y: number }> = null;
-
+    
+    public _onMouseMoved = new Observable<{ offsetX: number, offsetY: number }>();
+    public _moveCamera = true;
```
Comment (deltakosh) [inline src/Cameras/Inputs/freeCameraMouseInput.ts]: perhaps something like _allowCameraRotation would be clearer?

File: src/Cameras/Inputs/freeCameraMouseInput.ts [hunk 3]
```diff
@@ -31,8 +31,8 @@ export class FreeCameraMouseInput implements ICameraInput<FreeCamera> {
     private _observer: Nullable<Observer<PointerInfo>>;
     private previousPosition: Nullable<{ x: number, y: number }> = null;
     
-    public _onMouseMoved = new Observable<{ offsetX: number, offsetY: number }>();
-    public _moveCamera = true;
+    public onPointerMovedObservable = new Observable<{ offsetX: number, offsetY: number }>();
```
Comment (deltakosh) [inline src/Cameras/Inputs/freeCameraMouseInput.ts]: doc is missing I guess

Comment (TrevorDev) [reply src/Cameras/Inputs/freeCameraMouseInput.ts]: oops, i missed checking in a file 😡

**Pull request merged**